swarm/scheduler/filter.md

```diff
@@ -42,9 +43,9 @@ $ swarm manage --filter=health --filter=dependency
 
 ## Node filters
 
-When creating a container or building an image, you use a `constraint` or
-`health` filter to select a subset of nodes to consider for scheduling.
-If a node in Swarm cluster has a label with key `containerslots`
+When creating a container or building an image, you use a `constraint`,
+`whitelist` or `health` filter to select a subset of nodes to consider for
+scheduling.  If a node in Swarm cluster has a label with key `containerslots`
 and a number-value, Swarm will not launch more containers than the given number.
```

Comment:
Maybe "Swarm will only launch that number-value of containers."

Is `number-value` the literal parameter? Should we say `integer` instead of `number-value`?